.github/workflows/documentation.yml

```diff
@@ -14,17 +14,17 @@ on:
 
 jobs:
   build:
-    name: Build documentation
+    name: build
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v2
     - name: mkdocs build --strict
       run: |
         docker run --rm -v ${PWD}:/docs squidfunk/mkdocs-material build --strict --config-file docs/mkdocs.yml
   deploy:
-    name: Deploy documentation to gh-pages
+    name: deploy
     runs-on: ubuntu-20.04
-    if: github.event_name != 'pull_request'
+    if: github.event_name != 'push'
```

Comment:
Looks like you made a mistake here. This should be `== 'push'`. I'll tackle all this today.